.github/workflows/build.yml

```
@@ -0,0 +1,41 @@
+name: Build and SonarCloud Analysis
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)